src/main/java/com/dpgil/pathlinker/path_linker/internal/PathLinkerModel.java

```diff
@@ -626,14 +626,7 @@ private void checkAddEdge(HashMap<String, CyEdge> sourcetargetToEdge,
 	 * @param paths the list of paths generated by ksp algorithm
 	 */
 	private void createKSPSubgraph(ArrayList<Path> paths) {
```

Comment:
Update the comments and function to describe how the ksp subgraph is not created here, but the nodes and edges are selected here so the subgraph can be created by the `createKSPSubgraphAndView()` function in the `PathLinkerControlPanel` class